src/ui/public/stringify/types/Truncate.js

```diff
@@ -0,0 +1,31 @@
+define(function (require) {
+  return function TruncateFormatProvider(Private) {
+    var _ = require('lodash');
+    var FieldFormat = Private(require('ui/index_patterns/_field_format/FieldFormat'));
+
+    _.class(Truncate).inherits(FieldFormat);
+
+    function Truncate(params) {
+      Truncate.Super.call(this, params);
+    }
+
+    Truncate.id = 'truncate';
+    Truncate.title = 'Truncated String';
+    Truncate.fieldType = ['string'];
+
+    Truncate.prototype._convert = function (val) {
+      var length = this.param('fieldLength');
+      if (length > 0) {
+        return String(val).substr(0, length) + '...';
```

Comment:
This shouldn't be adding "..." to every value, only ones that are larger than the field length. I suggest you replace this with a call to [`_.trunc()`](https://lodash.com/docs#trunc)